microceph/cmd/microceph/cluster_list.go

```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"context"
+	"github.com/lxc/lxd/shared/i18n"
```

Comment:
The order of the imports should be revisited. The convention for the repository is:

```
built-in

external imports

internal imports (github.com/canonical/microceph/XYZ in this case)
```